homeassistant/components/binary_sensor/isy994.py

```diff
@@ -49,22 +68,154 @@ def setup_platform(hass, config: ConfigType,
 
 
 class ISYBinarySensorDevice(isy.ISYDevice, BinarySensorDevice):
-    """Representation of an ISY994 binary sensor device."""
+    """Representation of an ISY994 binary sensor device.
+
+    Often times, a single device is represented by multiple nodes in the ISY,
+    allowing for different nuances in how those devices report their on and
+    off events. This class turns those multiple nodes in to a single Hass
+    entity and handles both ways that ISY binary sensors can work.
+    """
 
     def __init__(self, node) -> None:
         """Initialize the ISY994 binary sensor device."""
         isy.ISYDevice.__init__(self, node)
+        self._child_nodes = []
```

Comment:
Where is this used?

Reply:
Hm, I had intended to store the child nodes in this array, but then I suppose I never actually ended up needing to. It feels weird not storing them, but since it would not be used in the current implementation I'll just remove this!